### PR TITLE
[pdmo-71]redesign the names and demographics components

### PR DESCRIPTION
**Pull request by CeciliaWanjiku**
# JIRA TICKET NAME:

PDMO-71: [Redesign the names and demographics component](https://issues.openmrs.org/browse/PDMO-71)

## SUMMARY:

Implement Feedback and change the display of the names & demographics component

### CODE REVIEW

File: app/js/components/demographics.jsx [hunk 1]
```diff
@@ -198,187 +209,205 @@ export default class Demographics extends React.Component {
...
+                                            <div className='form-group'>
+                                                <div className={this.state.causeOfDeathError ?
+                                                    'has-error' : ''
+                                                }>
+                                                    <label className='control-label col-sm-4'>
+                                                        Cause of death:
+                      </label>
+                                                    <div className='control-label col-sm-8'>
+                                                        <select className='form-control'
+                                                            onChange={this.handleChange}
```
Comment (jacksono) [inline app/js/components/demographics.jsx]: This closing label tag is out of indent

File: app/js/components/demographics.jsx [hunk 2]
```diff
@@ -198,187 +209,205 @@ export default class Demographics extends React.Component {
...
+                                                    value='option2'
+                                                    name='female'
+                                                    type='radio'
+                                                    value='F'
+                                                    checked={this.state.gender === 'F'}
+                                                    onChange={this.setGender}
+                                                /> Female
+                            </label>
+                                        </div>
+                                    </div>
```
Comment (jacksono) [inline app/js/components/demographics.jsx]: There are 2  closing label tags  out of indent

**Review by Farhanisto — COMMENTED** [review]
Looks fine

File: app/js/components/demographics.jsx [hunk 3]
```diff
@@ -35,7 +36,8 @@ export default class Demographics extends React.Component {
             format: 'YYYY-MM-DD HH:mm:ss',
             inputFormat: 'YYYY-MM-DD',
             mode: 'date',
-            deathDateTime: ''
+            deathDateTime: '',
+            showModal: false
```
Comment (Ewanjiru) [inline app/js/components/demographics.jsx]: Check into the indentation in this file. It seems to be more than two spaces

File: app/js/components/demographics.jsx [hunk 4]
```diff
@@ -198,187 +209,205 @@ export default class Demographics extends React.Component {
...
+                                                type='text'
+                                                value={this.state.createdBy + ' ' +
+                                                    this.state.dateCreated}
+                                            />
+                                        </div>
+                                    </div>
+                                </form>
+                            </ModalBody>
+                            <ModalFooter>
+                                <Button color="primary" type='submit' className='btn-success' onClick={this.handleSave}>Save</Button>{' '}
```
Comment (Ewanjiru) [inline app/js/components/demographics.jsx]: This line seems too long than required.

File: app/js/components/names.jsx [hunk 5]
```diff
@@ -133,120 +148,98 @@ class Name extends React.Component {
...
+                  />
+                </div>
               </div>
-            </div>
-          }
-        </form>
+            </form>
+          </ModalBody>
+          <ModalFooter>
+            <Button color="primary" type='submit' name='update' className='btn-success' onClick={this.handleSave}>Save</Button>{' '}
```
Comment (Ewanjiru) [inline app/js/components/names.jsx]: This line is too long.

File: app/js/components/names.jsx [hunk 6]
```diff
@@ -133,120 +148,98 @@ class Name extends React.Component {
...
+            <Button color="secondary" type='button' name='cancel' onClick={this.handleCancel}>Cancel</Button>
```
Comment (Ewanjiru) [inline app/js/components/names.jsx]: Ditto

File: app/js/components/demographics.jsx (hunk 1 above)
Comment (CeciliaWanjiku) [reply app/js/components/demographics.jsx]: @jacksono It is correctly indented. Just happens to be on its own line

File: app/js/components/demographics.jsx (hunk 2 above)
Comment (CeciliaWanjiku) [reply app/js/components/demographics.jsx]: Corrected. Thanks

File: app/js/components/demographics.jsx (hunk 4 above)
Comment (CeciliaWanjiku) [reply app/js/components/demographics.jsx]: Corrected. Thanks

File: app/js/components/names.jsx (hunk 5 above)
Comment (CeciliaWanjiku) [reply app/js/components/names.jsx]: Corrected

File: app/js/components/names.jsx (hunk 6 above)
Comment (CeciliaWanjiku) [reply app/js/components/names.jsx]: Corrected

File: app/js/components/demographics.jsx (hunk 1 above)
Comment (jacksono) [reply app/js/components/demographics.jsx]: 👍

**Pull request merged**